src/rust/engine/workunit_store/src/lib.rs

```diff
@@ -29,21 +29,25 @@ use concrete_time::TimeSpan;
 use parking_lot::Mutex;
 use rand::thread_rng;
 use rand::Rng;
+use std::collections::HashMap;
 use tokio::task_local;
 
 use std::cell::RefCell;
 use std::future::Future;
 use std::sync::Arc;
 
+pub type SpanId = String;
+
 #[derive(Clone, Debug, PartialEq, Eq, Hash)]
 pub struct WorkUnit {
```

Comment:
In Rust code, does this represent a CompletedWorkunit? If so, I recommend renaming.

Reply:
No, this represents the opaque ID we give to each workunit. It happens to be a random hex string, but doesn't need to be, hence the type alias.